app/src/main/java/com/teamagam/gimelgimel/app/GGApplication.java

```diff
@@ -44,6 +46,7 @@
     private ContainerMessagesViewModel mContainerMessagesViewModel;
     private MessageMapEntitiesViewModel mMessageMapEntitiesViewModel;
     private UsersLocationViewModel mUserLocationViewModel;
+    private Handler mBackgroundHandler;
```

Comment:
background is not informative enough.
consider change its name to receiversBackGroundThread.